documentation/release_notes.markdown

```diff
@@ -10,6 +10,14 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.11.1
+
+Released May 7 2020
+
+### Improvement
+
+- JRuby has been rolled back to 9.2.8.0 to attempt to mitigate an intermittent problem where some requests that go through JRuby error repeatedly with StackOverflow exceptions. [SERVER-2793](https://tickets.puppetlabs.com/browse/SERVER-2793)
```

Comment:
I'd either have this as a 'New feature', but remove "attempts to" —  "JRuby has been rolled back to 9.2.8.0 which mitigates an intermittent problem where some requests that go through JRuby error repeatedly with StackOverflow exceptions" (if this is confirmed). Or if not, maybe it should be a 'known issue'? And rephrased to "When doing XXX, an intermittent problem where some requests that go through JRuby error repeatedly with StackOverflow exceptions. JRuby has been rolled back to 9.2.8.0 in an attempt to mitigate this issue".